cmake/SetupCharm.cmake

```diff
@@ -86,3 +90,22 @@ configure_file(
 )
 
 include(SetupCharmModuleFunctions)
+
+# Make sure Charm++ was patched. If not you can get thread local storage errors
+# when loading the python bindings.
+if(NOT APPLE AND CHARM_VERSION VERSION_EQUAL 7.0.0 AND BUILD_PYTHON_BINDINGS)
+  # Check that the patch was applied:
+  set(_CHARM_CMAKE_FILE_TO_CHECK
+    "${CHARM_BINDIR}/../../cmake/detect-features-cxx.cmake")
```

Comment:
If you actually install charmc somewhere (as opposed to running it from the build directory) this file does not exist.

Reply:
I think you can detect this more reliably from `conv-mach-opt.sh` in the charm include directory.

Reply:
ah, yes, agreed!

Reply:
Pushed a fixup that checks the `conv` file instead